.github/workflows/hma-webapp-ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: hasher-matcher-actioner-webapp CI
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "hasher-matcher-actioner/webapp/**"
+      - ".github/workflows/hma-webapp-ci.yaml"
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "hasher-matcher-actioner/webapp/**"
+      - ".github/workflows/hma-webapp-ci.yaml"
+
+defaults:
+  run:
+    working-directory: hasher-matcher-actioner/webapp
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: yarn
+      - name: Run ESLint
+        run: eslint src
```

Comment:
Maybe related to the failure we are seeing, but perhaps `./node_modules/.bin/eslint` would be more appropriate.